index.html

```diff
@@ -20,11 +20,20 @@
 
       <div class="content">
 
+        <form action="information-about-book" method="POST">
+
+          <input type="text" name="name" value="Name" autofocus required>
+          <input type="text" name="author" value="Author" autofocus required>
+          <input type="text" name="price" placeholder="$" autofocus required>
```

Comment:
`autofocus` only works on one element, and that should probably be the _first_ element in the form